src/main/java/com/xeeva/catalog/ItemDetailsPage.java

```diff
@@ -1,221 +1,221 @@
+	@FindBy(xpath="//table/tbody/tr/td[2]/table/tbody/tr[5]/td/table/tbody/tr[1]/td[2]/input") 
+	private Textbox txtUnitPrice_RejectedOrderItem;
+	@FindBy(xpath="//table/tbody/tr/td[2]/table/tbody/tr[5]/td/table/tbody/tr[2]/td[2]/input") 
+	private Textbox txtQuantity_RejectedOrderItem;
+	@FindBy(xpath="//table[@id='tblItemDetails']/tbody/tr[2]/td[2]/a/div") 
+	private Button btnAddToCart_RejectedOrderItem;
+
+
+
+	/**Constructor**/
+	public ItemDetailsPage(OrasiDriver driver){
+		this.driver = driver;
+		ElementFactory.initElements(driver, this);
+	}
+
+	public void pageLoaded(){
+		catalogSearch.syncVisible(10, false);
+	}
+
+	/**Page Interactions**/
+
+	/**
+	 * @summary: Method to add Price Agreement Item to Cart.
+	 * @author: Praveen Namburi, @version: Created 08-09-2016
+	 */
+	public void addPriceAgrmntItemToCart(){
+		btnAddToCart.syncEnabled();
+		btnAddToCart.click();
+	}
+
+	/**
+	 * @summary Method to add non price agreement item to cart 
+	 * @author Lalitha Banda
+	 * @date 14/9/16
+	 **/
+	public void selectUOMValueAndAddNonPriceItemToCart(String strUOMValue){
+		if(lstSelectUOM.isDisplayed()){
+			lstSelectUOM.select(strUOMValue);
+			btnAddToCart.click();
+			driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
+			lblCartItemAddedMessage.syncVisible(20, false);
+			String getCartItemAddedMessage = lblCartItemAddedMessage.getText();
+			System.out.println("Message after adding item to cart : "+ getCartItemAddedMessage);
+			TestReporter.assertTrue(getCartItemAddedMessage.contains("added successfully!"), "Item added to the cart.");
+		}else{
+			TestReporter.logStep("Selected Random Item is Price aggrement Item!!");
+			btnAddToCart.jsClick();
+		}
+	}
+
+	/**
+	 * @summary: Method to Add-Item-To-Cart for Price agreement item.
+	 * @author: Praveen Namburi, @version: Created 12-09-2016
+	 */
+	public void addPriceAgreementItemToCart_And_Verify(){
+		btnAddToCart.jsClick();
+		driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
+		lblCartItemAddedMessage.syncVisible(15, false);
+		String getCartItemAddedMessage = lblCartItemAddedMessage.getText();
+		TestReporter.logStep("Message after adding item to cart : "+ getCartItemAddedMessage);
+		TestReporter.assertTrue(getCartItemAddedMessage.contains("added successfully!"), "Item added to the cart.");
+
+	}
+
+	/**
+	 * @summary Method to modify Item details for UOM,UP and quantity 
+	 * @author Lalitha Banda
+	 * @date 14/9/16
+	 **/
+
+	public void  modify_ItemDetails(String UP,String Qty,String UOM){
+		txtunitPrice.clear();
+		txtunitPrice.isDisplayed();
+		txtunitPrice.safeSet(UP);
+		txtQuantity.clear();
+		txtQuantity.safeSet(Qty);
+		lstunitofMeasure.select(UOM);
+	}
+
+
+	/**
+	 * @summary Method modify Item details - Global Search Records
+	 * @author Lalitha Banda
+	 * @date 14/9/16
+	 **/
+
+	public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMValue){
+		modify_ItemDetails(UnitPrice,Quantity,strUOMValue);
+		btnAddToCart.isDisplayed();
```

Comment:
If you are not doing anything with the return value of isDisplayed(), then please remove this line